deploy/monitoring/build/patch-kube-prometheus-stack.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta2
+kind: HelmRelease
+metadata:
+  name: kube-prometheus-stack
+  namespace: flux-system
+spec:
+  values:
+    alertmanager:
+      templateFiles:
+        cluster-name.tmpl: |-
+          {{ define "cluster_name" }}Build Cluster{{ end }}
+
+    grafana:
+      ingress:
+        annotations:
+          cert.gardener.cloud/issuer: ci-issuer
```

Comment:
These annotations are partly duplicated for both clusters. Should we move them to the base?

Reply:
I first thought about this too. It would save like 3 lines, but the ingress configurations are a bit harder to understand, because you have to look into two different files.
The base file is quite packed, so in my opinion it is easier to understand if you can see the ingress configuration at a glance.

Reply:
I understand your reasoning. Makes sense 👍